inventory.py

```diff
@@ -25,7 +28,10 @@
 except FileNotFoundError:
     logger.error('Configuration file could not be found; please add env.json to the config directory.')
 
-logging.basicConfig(level=ENV.get('LOG_LEVEL', 'DEBUG'))
+logging.basicConfig(level=ENV.get('LOG_LEVEL', 'DEBUG'),
+                    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
+                    handlers=[RotatingFileHandler('debug.log', mode='w', maxBytes=10485760,backupCount=5),
+                              logging.StreamHandler()])
```

Comment:
The reason I was using a `FileHandler` is I want to store logs for reviewing since the calls are async (the library is handling it ) I need to go back and read the logs to make sense. 
This shouldn't affect Openshift logging and shouldn't throw any error when Openshift disallow for creating a new file with `debug.log`. We can watch out for it this

Reply:
Does this need to be part of the application? Could you just use it for testing purposes?

Reply:
sounds like this won't work in OpenShift environment. If it serves purpose only for local testing, let's remove this from the pull request.

Reply:
Removed the handler